packages/cljfmt/package.yaml

```
@@ -0,0 +1,37 @@
+---
+name: cljfmt
+description: A tool for formatting Clojure code
+homepage: https://github.com/weavejester/cljfmt
+licenses:
+  - EPL-1.0
+languages:
+  - Clojure
+  - ClojureScript
+categories:
+  - Formatter
+
+source:
+  # renovate:versioning=loose
+  id: pkg:github/weavejester/cljfmt@0.12.0
+  asset:
+    - target: darwin_arm64
+      file: cljfmt-{{ version }}-darwin-aarch64.tar.gz
+      bin: cljfmt
+    - target: darwin_x64
+      file: cljfmt-{{ version }}-darwin-amd64.tar.gz
+      bin: cljfmt
+    - target: linux_arm64
+      file: cljfmt-{{ version }}-linux-aarch64.tar.gz
+      bin: cljfmt
+    - target: linux_x64_gnu
+      file: cljfmt-{{ version }}-linux-amd64.tar.gz
+      bin: cljfmt
+    - target: linux_x64
+      file: cljfmt-{{ version }}-linux-static-amd64.zip
+      bin: cljfmt
```

Great initiative including a GNU target! Generally static binaries are preferred, in order to avoid potential issues with libraries, specifically if the binary doesn't support "older" glibc versions. Since they provide one - let's just use that for all:

```suggestion
    - target: linux_x64
      file: cljfmt-{{ version }}-linux-static-amd64.zip
      bin: cljfmt
```